Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -221,21 +227,25 @@ public void onClick(View v) {
         mFloatingActionButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                if (!isAdded()) return;
-
-                addNote();
-                AnalyticsTracker.track(
-                        AnalyticsTracker.Stat.LIST_NOTE_CREATED,
-                        AnalyticsTracker.CATEGORY_NOTE,
-                        "action_bar_button"
-                );
+                createNewNote();
             }
         });
 
         getListView().setOnItemLongClickListener(this);
         getListView().setMultiChoiceModeListener(this);
     }
 
+    private void createNewNote(){
+        if (!isAdded()) return;
+
+        addNote();
+        AnalyticsTracker.track(
+                AnalyticsTracker.Stat.LIST_NOTE_CREATED,
+                AnalyticsTracker.CATEGORY_NOTE,
+                "action_bar_button"
```

Comment:
@roundhill Do you want this to be parameterized? For instance this will become `new_note_shortcut` for this new feature and `action_bar_button` for when the user taps on the fab.